crates/node/src/interactive_cli/context.rs

```diff
@@ -249,7 +339,21 @@ impl ContextCommand {
                     .invite_to_context(context_id, inviter_id, invitee_id)
                     .await?
                 {
-                    println!("{ind} Invited {} to context {}", invitee_id, context_id);
+                    if let Some(alias) = name {
+                        node.ctx_manager.create_alias(
+                            alias.clone(),
```

Comment:
`Alias<T>` impls `Copy`

```suggestion
                            alias,
```